recipes/exprtk/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "exprtk" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/ArashPartow/exprtk/archive/refs/tags/0.0.1.tar.gz
+  sha256: fb72791c88ae3b3426e14fdad630027715682584daf56b973569718c56e33f28
+  patches:
+    - 0001-add-conda_prefix.patch
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x' ) }}
+
+test:
+  commands:
+    - test -f $PREFIX/include/exprtk.hpp  # [unix]
+
+outputs:
+  - name: exprtk
+    script: build_exprtk.sh  # [unix]
+    requirements:
+      build:
+        - {{ compiler('c') }}
+        - {{ compiler('cxx') }}
+        - make  # [unix]
+
+about:
+  home: https://github.com/ArashPartow/exprtk
```

Comment:
Can this be changed to: 

https://www.partow.net/programming/exprtk/index.html

Reply:
Sure!
I'll do it right away. :)
Sorry, I just realized my reply to your email didn't go through.